listenbrainz/spark/handlers.py

```diff
@@ -342,3 +343,23 @@ def handle_similar_users(message):
             from_name='ListenBrainz',
             from_addr='noreply@'+current_app.config['MAIL_FROM_DOMAIN'],
         )
+
+
+def handle_similar_users_2021(message):
```

Comment:
Hmm. The year in this function name doesn't feel future proof to me. I guess we will keep and reuse this for next year, no? handle_similar_users_year_end() perhaps?

Reply:
Done.